modules/oe_migration_views/oe_migration_views.views.inc

```diff
@@ -0,0 +1,480 @@
+<?php
+
+/**
+ * @file
+ * Contains hooks for Views.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\migrate\Exception\RequirementsException;
+use Drupal\migrate\Plugin\RequirementsInterface;
+use Drupal\oe_migration_views\Plugin\migrate\id_map\SqlData;
+use Drupal\migrate\Plugin\migrate\destination\EntityContentBase;
+use Drupal\migrate\Plugin\migrate\id_map\Sql;
+use Drupal\migrate\Plugin\Migration;
+
+/**
+ * Implements hook_views_data_alter().
+ */
+function oe_migration_views_views_data_alter(&$data) {
+  $plugin_manager = \Drupal::service('plugin.manager.migration');
+  $migrations = $plugin_manager->createInstances([]);
```

Comment:
If the source database is not installed/accessible, as it actually happens with the containers in acceptance and prod, it will through two exceptions and will stop the execution of the pipeline.
```
In Connection.php line 420:
  SQLSTATE[HY000] [1049] Unknown database '[d7 database]'  

In Connection.php line 416:
  SQLSTATE[HY000] [1049] Unknown database '[d7 database]' 
```